components/array-object-table/src/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Table } from 'govuk-frederic';
+
+import { rowsFromArray, titlesFromFields } from '@govuk-frederic/utils';
+
+/**
+ *
+ * ### Usage
+ *
+ * Simple
+ * ```jsx
+ * const fields = [
+ *   { key: 'one', heading: 'One' },
+ *   { key: 'two', heading: 'Two' },
+ * ];
+ * const array = [
+ *   { one: 'test', two: 'test' },
+ *   { one: 'test' },
+ *   {},
+ * ];
+ * const title = ['Heading'];
+ * 
+ * <ArrayObjectTable fields={fields} array={array} title={title}/>;
+ * ```
+ * 
+ * With skipEmptyRows
+ * ```jsx
+ * const fields = [
+ *   { key: 'one', heading: 'One' },
+ *   { key: 'two', heading: 'Two' },
+ * ];
+ * const array = [
+ *   {},
+ *   {},
+ * ];
+ * const title = ['Heading'];
+ * 
+ * <ArrayObjectTable fields={fields} array={array} title={title} skipEmptyRows/>
+ * ```
+ * 
+ * With skipEmptyRows and hideWithNoValues
+ * ```jsx
+ * const fields = [
+ *   { key: 'one', heading: 'One' },
+ *   { key: 'two', heading: 'Two' },
+ * ];
+ * const array = [
+ *   {},
+ *   {},
+ * ];
+ * const title = ['Heading'];
+ * 
+ * <ArrayObjectTable fields={fields} array={array} title={title} skipEmptyRows hideWithNoValues/>;
+ * ```
+ * */
+const ArrayObjectTable = ({ fields = [], array = [], hideWithNoValues = false, skipEmptyRows = false, title, ...props }) => {
```

Comment:
might be better/nicer to use `defaultProps` for `hideWithNoValues` and `skipEmptyRows`